src/Parser/Priest/Discipline/Normalizers/ShadowfiendNormalizer.js

```diff
@@ -0,0 +1,20 @@
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+import SPELLS from 'common/SPELLS';
+
+class ShadowfiendNormalizer extends EventsNormalizer {
+
+  normalize(events) {
+    events.forEach((event, eventIndex) => {
+      if (event.type === "cast") {
+        const spellId = event.ability.guid;
+        if (spellId === SPELLS.SHADOWFIEND_WITH_GLYPH_OF_THE_SHA.id || spellId === SPELLS.LIGHTSPAWN.id) {
+          event.ability.guid = SPELLS.SHADOWFIEND.id;
```

Comment:
When modifying events please set the `event.__modified` prop to `true` whenever possible. I suggest also including the original spellId in `oldGuid` or something like that.